.gitignore

```diff
@@ -4,4 +4,5 @@ node_modules
 .nyc_output/
 coverage/
 npm-debug.log
-package-lock.json
\ No newline at end of file
+package-lock.json
+.vscode
```

Comment:
Don't forget to add `.vscode` to [your global gitignore file](https://help.github.com/articles/ignoring-files/) as well.

Reply:
Will do!